src/Psalm/CallMap.php

```diff
@@ -7719,7 +7719,7 @@
 'Reflection::getModifierNames' => ['array', 'modifiers'=>'int'],
 'ReflectionMethod::__construct' => ['void', 'class'=>'string|object', 'name'=>'string'],
 'ReflectionMethod::__construct\'1' => ['ReflectionMethod', 'class_method'=>'string'],
-'ReflectionMethod::export' => ['?string', 'class'=>'string', 'name'=>'string', 'return='=>'bool'],
+'ReflectionMethod::export' => ['string', 'class'=>'string', 'name'=>'string', 'return='=>'bool'],
```

Comment:
The docs are wrong here – this returns `null` unless a third arg of `true` is given

Reply:
Addressed both comments, incorrectly copied the incorrect line here